assets/dev/scss/editor/preview/preview.scss

```diff
@@ -18,7 +18,7 @@
 @import "inline-editor";
 
 @import "../../../../../modules/elements-color-picker/assets/scss/elements-color-picker";
-@import "../../../../../modules/floating-buttons/assets/scss/preview/floating-buttons-preview";
+@import "../../../../../modules/floating-buttons/assets/scss/preview/contact-buttons-preview";
```

Comment:
Are we sure this needs to be renamed? It looks like it's actually related to the Floating Buttons Document and not widget specific.

Reply:
@louiswol94 agreed and changed